selenium/src/main/scala/not/ogame/bots/selenium/ParsingUtils.scala

```diff
@@ -1,12 +1,16 @@
 package not.ogame.bots.selenium
 
-import java.time.{Instant, ZoneOffset, ZonedDateTime}
+import java.time.ZonedDateTime
 
 import not.ogame.bots.{Coordinates, LocalClock}
 
 object ParsingUtils {
   /** @param coordinatesText Coordinates test in "[1:1:1]" format. */
   def parseCoordinates(coordinatesText: String): Coordinates = {
+    if (coordinatesText.length < 5) {
+      //Happens when returning from destroyed moon
+      return Coordinates(0, 0, 0)
```

Comment:
Shouldn't we somehow express this coordinates as invalid?

Reply:
How about leaving it as TODO forever. It is really rare case and I don't want to pollute our code base with one more option where we will be always calling blindly get. Even when bots will start sending fleets on Destroy mission there would be no use for option here.